sklearnex/tests/test_config.py

```diff
@@ -28,16 +28,96 @@ def test_get_config_contains_sklearn_params():
 
 
 def test_set_config_works():
+    """Test validates that the config settings were applied correctly by
+    set_config.
+    """
```

Comment:
These tests will require generalization and parameterization in future.

Reply:
Thank you @Alexsandruss . Please suggest inputs I will cover with a ticket